services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/table/metadata/providers/Constants.java

```diff
@@ -37,6 +37,8 @@ public class Constants {
 	};
 
 	static final ObjectField[] DATASET_DEFAULT_COLUMNS = new ObjectField[]{
```

Comment:
DATASET_DEFAULT_COLUMNS  -> DATASET_SPECIFIC_COLUMNS

Also rename the FILE_DEFAULT_COLUMNS -> FILE_SPECIFIC_COLUMNS
Remove from the files everything that is in default.